src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -338,21 +261,21 @@ private void fetchUserInfo() {
   }
 
   /**
-   * Fetch the workspace metadata (if it's not already loaded) and the pet SA credentials. Don't
-   * throw an exception if it fails, because that shouldn't block a successful login, but do log the
-   * error to the console.
+   * Fetch the workspace metadata (if it's not already loaded) and pet SA email. Don't throw an
+   * exception if it fails, because that shouldn't block a successful login, but do log the error to
+   * the console.
    */
   private void fetchWorkspaceInfo() {
     try {
       if (!Context.requireWorkspace().getIsLoaded()) {
-        // if the workspace was set without credentials, load the workspace metadata and pet SA
+        // if the workspace was set without credentials, load the workspace metadata
         Workspace.load(Context.requireWorkspace().getId());
       } else {
         // otherwise, just load the pet SA
-        fetchPetSaCredentials();
+        fetchPetSaEmail();
       }
     } catch (Exception ex) {
-      logger.error("Error loading workspace or pet SA credentials during login", ex);
+      logger.error("Error loading workspace or pet SA email during login", ex);
       UserIO.getErr()
           .println(
               "Error loading workspace information for the logged in user (workspace id: "
```

Comment:
As a user, I'm not sure what I'd do with this message. Maybe indicate that the login continued despite the error?

Reply:
I'd have to see this error in the wild, to know what a better error message would be

Reply:
I don't know if we have user-facing docs about pet SAs anywhere, but I suppose they'll call us if there's trouble.